deploy/kubernetes/kubectl/wsnext/Readme.md

```diff
@@ -2,23 +2,24 @@
 
 Execute 
 ```shell
-docker build --no-cache -t garagatyi/che-marketplace:1 .
+docker build --no-cache -t garagatyi/che-marketplace:os .
 ```
 Where `--no-cache` is needed to prevent usage of cached layers with marketplace files.
 Useful when you change marketplace files and rebuild the image.
 
-`-t garagatyi/che-marketplace:1` is needed to set identifier of the image. Do not forget to change `feature-api/yaml` file accordingly if you change this identifier to something else.
+`-t garagatyi/che-marketplace:os` is needed to set identifier of the image.
+Do not forget to change `feature-api.yaml` file accordingly if you change this identifier to something else.
```

Comment:
Now we have editor and plugins. Maybe we should rename `feature-api.yaml` to something different. Because we don't have features anymore.

Reply:
I don't want to change it because there is fresh blog post from Mario that uses instructions where this name is used.

Reply:
oook. however, it looks weird. For how long are you going to keep it in this way?